client/components/mma/savedArticles/SavedArticles.tsx

```diff
@@ -1,13 +1,61 @@
+import { typeCheckObject } from '../../../../shared/typeCheckObject';
+import { fetchWithDefaultParameters } from '../../../utilities/fetch';
+import {
+	LoadingState,
+	useAsyncLoader,
+} from '../../../utilities/hooks/useAsyncLoader';
+import { GenericErrorScreen } from '../../shared/GenericErrorScreen';
 import { NAV_LINKS } from '../../shared/nav/NavConfig';
 import { PageContainer } from '../Page';
+import { JsonResponseHandler } from '../shared/asyncComponents/DefaultApiResponseHandler';
+import { isSavedArticlesResponse } from './models/SavedArticle';
+import type { SavedArticlesResponse } from './models/SavedArticle';
+import { SavedArticlesDisplay } from './SavedArticlesDisplay';
+import { SavedArticlesEmpty } from './SavedArticlesEmpty';
+import { SavedArticlesLoading } from './SavedArticlesLoading';
 
 export const SavedArticles = () => {
 	return (
 		<PageContainer
 			selectedNavItem={NAV_LINKS.savedArticles}
 			pageTitle={NAV_LINKS.savedArticles.title}
 		>
-			<p>There is nothing to see here yet</p>
+			<SavedArticlesPage
+				saveForLaterAPICall={() =>
+					fetchWithDefaultParameters('/add/url')
+				}
+			/>
 		</PageContainer>
 	);
 };
+
+interface SavedArticlesPageProps {
+	saveForLaterAPICall: () => Promise<unknown>;
+}
+
+export function SavedArticlesPage(props: SavedArticlesPageProps) {
```

Comment:
Wonder if it's worth looking at the AsyncLoader utility used by the AccountOverview? There seem to be two async loaders in use in this repo - useAsyncLoader hook and the AsyncLoader class compononent. I would check with the rest of the MMA team which component is preferred and see if we can align our work with that.